### PR TITLE
Add informative content

### PR DESCRIPTION
**Pull request by OR13**
describe general types of wallets and cloud secret managers

### CODE REVIEW

**Review by kimdhamilton — APPROVED** [review]

**Comment by kimdhamilton** [comment]
pinging @NTonani and @mprorock for review; otherwise I think we can merge EOD tomorrow

File: docs/index.html [hunk 1]
```diff
@@ -221,6 +221,89 @@ <h2>Introduction</h2>
...
+          Progressive web applications (PWAs) are a kind of offline capable website.
+        </p>
+
+        <p>
+          You can read more about them 
+          <a href="https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps">here</a>.
+        </p>
+
+        <p>
+          PWA's don't require users to install an app throught an app store,
```
Comment (TallTed) [inline docs/index.html:243]:
```suggestion
          PWAs don't require users to install an app through an app store, 
```

File: docs/index.html [hunk 2]
```diff
@@ -221,6 +221,89 @@ <h2>Introduction</h2>
...
+        </p>
+
+        <p>
+          PWA's don't require users to install an app throught an app store, 
+          but are limited by browser API support in ways that native mobile applications are not.
+        </p>
+
+        <p>
+          PWA's can access camera, webrtc, nfc, bluetooth and other browser apis. 
+          They can connect to remote api's to facilitate discovery and messaging.
```
Comment (TallTed) [inline docs/index.html:249]:
```suggestion
          PWAs can access camera, WebRTC, NFC, Bluetooth, and other browser APIs. 
          They can connect to remote apis to facilitate discovery and messaging.
```

File: docs/index.html [hunk 3]
```diff
@@ -221,6 +221,89 @@ <h2>Introduction</h2>
...
+        </p>
+      </section>
+
+      <section>
+        <h3>Native Mobile Applications</h3>
+        <p>
+          Native Mobile Applications are software a user installs on a mobile OS, like Android or iOS.
```
Comment (TallTed) [inline docs/index.html:256]:
```suggestion
          Native Mobile Applications (NMAs) are software a user installs on a mobile OS, like Android or iOS.
```

File: docs/index.html [hunk 4]
```diff
@@ -221,6 +221,89 @@ <h2>Introduction</h2>
...
+        </p>
+
+        <p>
+          Native mobile applications have accesss to APIs supported by the mobile OS, 
+          which might include filesystem, networking, phone, or camera APIs.
```
Comment (TallTed) [inline docs/index.html:261]:
```suggestion
          NMAs have access to APIs supported by the mobile OS, such as 
          filesystem, network, phone, or camera APIs, among others.
```

File: docs/index.html [hunk 5]
```diff
@@ -221,6 +221,89 @@ <h2>Introduction</h2>
...
+        </p>
+
+        <p>
+          Native mobile applications might implement support for hardware backed secret or key management via the OS.
+          This and better biometric authentication support make them attractive to high security applications.
```
Comment (TallTed) [inline docs/index.html:266]:
```suggestion
          NMAs might implement support for hardware backed secret or key 
          management via the OS. This and better biometric authentication 
          support make them attractive to high security applications.
```

File: docs/index.html [hunk 6]
```diff
@@ -221,6 +221,89 @@ <h2>Introduction</h2>
...
+        <p>
+          Cloud Wallets are software services hosted on the internet, typically exposed over HTTP or gRPC.
+        </p>
+
+        <p>
+          Cloud Wallets offer some of the security benefits of centralization, such as logging and dynamic scaling.
+        </p>
+
+        <p>
+          Cloud wallets can also be "always online", in cases where a mobile wallet might loose connectivity.
```
Comment (TallTed) [inline docs/index.html:281]:
```suggestion
          Cloud wallets can also be "always online", in cases where a mobile wallet might lose connectivity.
```

File: docs/index.html [hunk 7]
```diff
@@ -221,6 +221,89 @@ <h2>Introduction</h2>
...
+        </p>
+
+        <p>
+          Cloud wallets and message based APIs are generally an easier integration target with legacy systems.
```
Comment (TallTed) [inline docs/index.html:285]:
```suggestion
          Cloud wallets and message based APIs are generally an easier integration target for legacy systems.
```

File: docs/index.html [hunk 8]
```diff
@@ -221,6 +221,89 @@ <h2>Introduction</h2>
...
+        </p>
+      </section>
+
+      <section>
+        <h3>Cold Storage Wallets</h3>
+        <p>
+          Cold storage wallets are non networked long term storage for digital assets.
```
Comment (TallTed) [inline docs/index.html:292]:
```suggestion
          Cold Storage Wallets (CSWs) are non-networked, long-term storage for digital assets.
```

File: docs/index.html [hunk 9]
```diff
@@ -221,6 +221,89 @@ <h2>Introduction</h2>
...
+        </p>
+
+        <p>
+          Cold storage is typcially reserved for very sensitive information, 
+          such as keys that control large amounts of crypto currency, or root certificate signing keys.
```
Comment (TallTed) [inline docs/index.html:297]:
```suggestion
          Cold storage is typically reserved for very sensitive information, 
          such as keys that control large amounts of crypto currency, or root 
          certificate signing keys.
```

File: docs/index.html [hunk 10]
```diff
@@ -221,6 +221,89 @@ <h2>Introduction</h2>
...
+        </p>
+
+        <p>
+          Cold storage wallets may be implemented as either isolated hardware or physical paper.
```
Comment (TallTed) [inline docs/index.html:301]:
```suggestion
          CSWs may be implemented as either isolated hardware or physical paper.
```

File: docs/index.html [hunk 11]
```diff
@@ -1602,6 +1685,29 @@ <h4>Google Wallet</h4>
...
         <a href="http://wallet.microsoft.com/">wallet.microsoft.com</a>
       </section>
+
+      <section>
+        <h3>Cloud Secret Managers</h3>
+
+        <p>Software services that support secret or key management are useful building blocks for cloud wallets.</p>
+
+        <p class="note">Some services combine secret and key operations under the ombrella term "Key Management Service". 
+          Wallet's may use both managed asymmetric or symmetric key operations as well as managed and versioned secrets.
```
Comment (TallTed) [inline docs/index.html:1695]:
```suggestion
        <p class="note">
          Some services combine secret and key operations under the 
          umbrella term "Key Management Service".  Wallets may use both 
          managed asymmetric and symmetric key operations, as well as 
          managed and versioned secrets.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Mostly typo fixes, some line length tweaks, increased consistency of acronym use...

**Review by NTonani — APPROVED** [review]
_Very_ informative content!

**Comment by OR13** [comment]
@TallTed Thank you!

**Pull request merged**